readthedocs/api/v2/utils.py

```diff
@@ -212,9 +212,9 @@ def delete_versions_from_db(project, tags_data, branches_data):
         .exclude(active=True)
     )
     _, deleted = to_delete_qs.delete()
-    versions_count = deleted.get('builds.Version')
+    versions_count = deleted.get('builds.Version', 0)
```

Comment:
If the default is now `0`, doesn't that mean that `%d` can be kept? Or am I missing something?

Reply:
I added the 0 as default to avoid None and render it nicely (eg. `versions_count=0` instead of `versions_count=None`). I'm using `%s` to avoid the same issue we hit here in the feature because the usage of `%d`.

Technically speaking, `%d` is correct _and_ better (?). Practically speaking, `%s` doesn't break because something dumb as this 😄